stable_examples/examples/std_global_allocator.rs

```diff
@@ -1,20 +1,21 @@
 use talc::*;
 
-// note: miri thinks this violates stacked borrows upon program termination.
-// This only occurs if #[global_allocator] is used.
-// Use the allocator API if you can't have that. Perhaps checck out `std_allocator_api.rs`?
+// note:
+// - Miri thinks this violates stacked borrows upon program termination.
+//   - This only occurs with `#[global_allocator]`.
+//   - Use the allocator API if you can't have that. Perhaps check out `examples/stable_allocator_api.rs`?
```

Comment:
Alternative, although it'd be good to know what "if you can't have that" is referring to.

```suggestion
//   - Consider using the allocator API if you can't have that (see: `examples/stable_allocator_api.rs`)
```